lib/settings.js

```diff
@@ -183,9 +215,54 @@ function init ( ) {
     return enabled;
   }
 
+  function isAlarmEventEnabled (notify) {
+    var enabled = false;
+
+    if (!notify.eventName) {
+      enabled = true;
+    } else if (notify.eventName === 'high' && notify.level === levels.URGENT && settings.alarmUrgentHigh) {
+      enabled = true;
+    } else if (notify.eventName === 'high' && settings.alarmHigh) {
+      enabled = true;
+    } else if (notify.eventName === 'low' && notify.level === levels.URGENT && settings.alarmUrgentLow) {
+      enabled = true;
+    } else if (notify.eventName === 'low' && settings.alarmLow) {
+      enabled = true;
+    }
+
+    return enabled;
+  }
+
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/941995763/issues/source?bid=2452379&fileBranchId=2847143#l236)